xbmc/playlists/PlayListTypes.h

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+namespace PLAYLIST
+{
+
+using Id = int;
```

Comment:
dont wanna be your pita but i think `Id` is an unfortune choice here. at least to me it implies there can be multiple playlists with id‘s 1-x. can this identifier be changed to `Type` ? i even volunteer changing it after this pr is in. np

Reply:
playlists was originally id e.g. jsonrpc methods call them "playlistid"
which is one of reason why i kept the name "id"
if you want i will apply a rename of all variables (except for methods like jsonrpc)

the important thing to me is that others team membr give feedback here,
I would not like to make hundreds of changes and then go back

Reply:
sure only if it makes sense. (i still volunteer) lets wait for opinions